configs/module/network/birdnet_inference.yaml

```diff
@@ -1,6 +1,6 @@
 model: 
   _target_: birdset.modules.models.birdnet.BirdNetModel
-  num_classes: ${datamodule.dataset.n_classes}
```

Comment:
@XgamerTV Could we revert this change or does this than break something? Haven't we concluded that this will be overridden in train.py anyway? But there might be use cases why the line was like it is.

Reply:
@XgamerTV You deleted the line:
```
num_classes: ${datamodule.dataset.n_classes}
```
and added the line:
```
num_classes: null
```
Instead.